bin/executables/lib/playground.rb

```diff
@@ -0,0 +1,45 @@
+module AlephExecutables
+  class Playground
+    def self.setup(host, db, port, user, password, options = {})
+      config_path = options[:config_path] || '/tmp/aleph/demo/configuration'
+
+      seed_db = options[:seed_db]
+
+      config_generator = ConfigGenerator.new(config_path, 'playground')
+
+      # redis envs
+      config_generator.merge_envs(redis_url: 'redis://localhost:6379')
```

Comment:
i mean its all happening in playground.rb now, aka the actual values are specified there. and the config_generator is just a tool which writes the specified configs and and envs